ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -11007,6 +11007,16 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f21-destructive
+  reporter_config:
+    slack:
+      channel: '#forum-prow-hypershift-qe-ci'
```

Comment:
change to 
```
      job_states_to_report:
      - failure
      - error
```

Reply:
@LiangquanLi930 I think we should add success job, b/c some success jobs also has failed cases. For aborted and pending we can remove. WDYH?

Reply:
ack

Reply:
```
      job_states_to_report:
      - failure
      - error
      - success
```